### ISSUE TITLE
create new credential - whole process

### ISSUE THREAD

**Issue by veitmo** [amosproj/amos2022ss04-digital-identity#113]
## User story

1. As a {HR-Employee}
2. I want / need {to create a new credential which I can later assign to a DI}
3. So that {I can create a Mitarbeiterausweis for example}

## Acceptance criteria
* process of credential creation running successfully:
  * new credential-information can be entered in frontend #112 
  * backend takes information from frontend and sends request to Lissi API to create credential and receives response #111 
  * feedback on success/error is displayed in frontend

## Definition of done (DoD)

- Added only after week 5
- The same for all features
- Here goes the project specific part

## DoD general criteria

- Feature has been fully implemented
- Feature has been merged into the mainline
- All acceptance criteria were met
- Product owner approved features
- All tests are passing
- Developers agreed to release
